File: Backend/TaskManager/ViewModels/TaskViewModel.cs
namespace TaskManager.ViewModels;

public class TaskViewModel
{
    public string? id { get; set; }
    public string? title { get; set; }
    public string? description { get; set; }
    public DateTime expirationTime { get; set; }
    public DateTime createdTime { get; set; }
}

File: Frontend/TaskManager.Front/src/viewmodels/TaskViewModel.ts
export class TaskViewModel {
    id: string
    title: string
    description: string
    expirationTime : Date
    createdTime : Date

    isImportant: boolean

    constructor() {
        this.id = ''
        this.title = ''
        this.description = ''
        this.expirationTime = new Date()
        this.createdTime = new Date()
        this.isImportant = false
    }
}